AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/SubscriptionSync/ModelReconciliationQueueBehaviorTests.swift

```diff
@@ -89,9 +89,34 @@ class ModelReconciliationQueueBehaviorTests: ReconciliationQueueTestBase {
             subscriptionEventsSubject.send(.mutationEvent(mutationSync))
         }
 
+        let eventsSentViaPublisher1 = expectation(description: "id-1 sent via publisher")
+        let eventsSentViaPublisher2 = expectation(description: "id-2 sent via publisher")
+        let eventsSentViaPublisher3 = expectation(description: "id-3 sent via publisher")
+        let queueSink = queue.publisher.sink(receiveCompletion: { _ in
+            XCTFail("Not expecting a call to completion")
+        }, receiveValue: { event in
+            if case let .mutationEvent(mutationEvent) = event {
+                switch mutationEvent.modelId {
+                case "id-1":
+                    eventsSentViaPublisher1.fulfill()
+                case "id-2":
+                    eventsSentViaPublisher2.fulfill()
+                case "id-3":
+                    eventsSentViaPublisher3.fulfill()
+                default:
+                    XCTFail("Not expecting a call to default")
+                }
+            }
+        })
+
         queue.start()
 
-        wait(for: [event1Saved, event2Saved, event3Saved], timeout: 5.0, enforceOrder: true)
+        wait(for: [event1Saved,
+                   event2Saved,
+                   event3Saved], timeout: 5.0, enforceOrder: true)
```

Comment:
nit: format. Make sure to run swiftlint & swiftformat. If they both pass this, well then, we should revise those rules :)

Reply:
looks like we'll need to revise the rules.